qiskit/transpiler/passes/routing/commuting_2q_gate_routing/commuting_2q_block.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, node_block: Iterable[DAGOpNode]) -> None:
             )
 
         super().__init__(
-            "commuting_2q_block", num_qubits=len(qubits), params=[], label="Commuting 2q gates"
+            "commuting_2q_blocks", num_qubits=len(qubits), params=[], label="Commuting 2q gates"
```

Comment:
This comes from https://github.com/Qiskit/qiskit-terra/pull/7813#discussion_r856217819

I actually think the name of the class should be the one changing. Is a single block that contains 2q gates that commute.

Reply:
Done here: [401b07c](https://github.com/Qiskit/qiskit-terra/pull/7979/commits/401b07c6ea0283f3ebb0f6ca71a3ff1607271470)